### DISCUSSION TITLE
Can't use Cmd-F to search for text displayed in the browser

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [sourcelair/xterm.js#4548] [Q&A] [answered]
## Details
- Browser and browser version: chrome Version 113.0.5672.126 (Official Build) (x86_64)
- OS version: mac
- xterm.js version: 5.1.0

### Steps to reproduce

1. Start a terminal with any output
2. Use Cmd-F to search for text, the browser prompts no match

**Comment by Tyriar** [comment, marked as answer]
This is by design, you need to use the [search addon](https://github.com/xtermjs/xterm.js/tree/master/addons/xterm-addon-search) for this.

**Reply by CaledoniaProject** [reply]
Does it mean I need to implement my own search dialog as well?

**Reply by Tyriar** [reply]
Yes, the addon exposes calls your dialog would call
